src/Shared/Targets/Wrappers/AspNetBufferingTargetWrapper.cs

```diff
@@ -209,12 +209,37 @@ protected virtual void SaveHttpContext(HttpContextEventArgs httpContextEventArgs
 #endif
         }
 
+        private bool _firstWrite = true;
```

Comment:
Every WebRequest will have their own HttpContext-scope, so not sure this makes any sense.

Was thinking `GetRequestBuffer` could be called `GetOrCreateBuffer`

Reply:
I see, yes i will change that

Reply:
Fixed as suggested